draft-ietf-mls-architecture.md

```diff
@@ -1430,12 +1430,12 @@ deniability requires further analysis.
 
 ### Associating a User's Clients
 
-When the same user uses multiple clients, it may be possible for other members
-of a group to recognize all of those clients as belonging to the same user.  For
-example, all of a user's clients might present credentials authenticating the
-user's identity.  This association among devices might be considered a leak of
-private information.  The remainder of this section describes several approaches
-for addressing this.
+When a user has multiple devices, the base MLS protocol requires each device to
```

Comment:
```suggestion
When a user has multiple devices, the base MLS protocol expects each device to
```
?

Reply:
No, it simply has no opinion on the topic.